app/controllers/sicAndCompliance/ComplianceExitController.scala

```diff
@@ -37,14 +36,17 @@ class ComplianceExitController (ds: CommonPlayDependencies)(implicit vrs: Regist
       _ <- vrs.submitSicAndCompliance()
     } yield Redirect(controllers.vatFinancials.vatBankAccount.routes.CompanyBankAccountController.show())
 
-
   def clearComplianceContainer(implicit hc: HeaderCarrier): Future[S4LVatSicAndCompliance] =
-    viewModel[BusinessActivityDescription]().
-      fold(S4LVatSicAndCompliance())(bad => S4LVatSicAndCompliance(description = Some(bad)))
+    for {
+      bad <- viewModel[BusinessActivityDescription]().
+        fold(Option.empty[BusinessActivityDescription])(Some(_))
```

Comment:
Don't think it's a Some into a Some - it's a BusinessActivityDescription into a Some